### PR TITLE
Update dependencies and move to ember@~2.8.0

### PR DESCRIPTION
**Pull request by dafortin**
#CHANGELOG
* **Update** dependencies (engine, ember-cli, ember-data)

### CODE REVIEW

File: addon/controllers/new.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default Ember.Controller.extend({
     post (title, body) {
       let blog = this.get('blog')
       blog.add(title, body)
-      this.transitionToRoute('/blog')
+      this.transitionToRoute('index')
```
Comment (rox163) [inline addon/controllers/new.js:9]: why this change?

Comment (dafortin) [reply addon/controllers/new.js:9]: I'm getting the following error: Route#transitionTo cannot be used for URLs. Please use the route name instead.

Comment (rox163) [reply addon/controllers/new.js:9]: ah I see. and there is no route named `blog`? just wondering how it affects the final behaviour if at all. maybe if there is a route name blog then you just take the slash `/` away?

Comment (dafortin) [reply addon/controllers/new.js:9]: I did try that and no there is no route name blog. /blog was going back to the index.

**Comment by rox163** [comment]
Thanks @srowhani
